### PR TITLE
Release RPM 4.18.1

### PR DESCRIPTION
**Pull request by dmnks**

### CODE REVIEW

File: wiki/Releases/4.18.1.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+## Download information
+ * Source: [rpm-4.18.1.tar.bz2](https://ftp.osuosl.org/pub/rpm/releases/rpm-4.18.x/rpm-4.18.1.tar.bz2)
+ * SHA256SUM: 37f3b42c0966941e2ad3f10fde3639824a6591d07197ba8fd0869ca0779e1f56
+
+## Summary of changes from RPM 4.18.0
+
+### General bugfixes and enhancements
+#### Command line
+* Fix signature reserved space not restored on signing (#2382, regression in 4.15.0)
```
Comment (pmatilai) [inline wiki/Releases/4.18.1.md]: I think the issue here was not restoring the space when deleting a signature?

**Comment by pmatilai** [comment]
Other than that, looks fine to me.

File: wiki/Releases/4.18.1.md (hunk 1 above)
Comment (dmnks) [reply wiki/Releases/4.18.1.md]: Ugh, you're right, this was a failed attempt at rewording these items to more "uniformly" talk about signing in general, but this particular commit was about deleting. Fixing now, thanks!

**Comment by dmnks** [comment]
Thanks, I'll merge this then.

**Pull request merged**